Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,83 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local IFS=$'\n'
+  local basedir="${1/\~/$HOME}"
```

Comment:
This seems like it's going to DTWT on `~foo` and `foo~bar`.  Could you please make it more robust?  For that matter, why isn't `local basedir="$1"` good enough?

Reply:
I don't know exactly why I had this, but with `_values -C "packages from $basedir" $basedir/*(-/N:t)`, `local basedir="$1"` seems sufficient. So, I'm going to make a modification in that way.